### PR TITLE
Feature: add development instructions and recompile dist module

### PR DESCRIPTION
**Pull request by artemii-yanushevskyi**
* Add development instructions for contributors
* Recompile `dist` module
* Add a license file to `dist` module

The project configuration is now compatible with yarn 3.6.0

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,7 +6,6 @@
   "repository": "git@github.com:codemagic-ci-cd/trigger-codemagic-workflow-action.git",
   "author": "Artemii Yanushevskyi <artemii@nevercode.io>",
   "license": "MIT",
-  "private": false,
```
Comment (artemii-yanushevskyi) [inline package.json:9]: Removed by yarn

File: README.md [hunk 2]
```diff
@@ -68,3 +68,15 @@ Output variables can be used later in the action steps:
...
+Development
+-----------
+
+Install dependencies
+
+    yarn
+
+Complie sources and dependencies into a standalone module
+
+    yarn ncc build index.js --license LICENSE
```
Comment (artemii-yanushevskyi) [inline README.md]: https://docs.github.com/en/actions/creating-actions/creating-a-javascript-action

File: package.json [hunk 3]
```diff
@@ -17,4 +16,3 @@
     "@vercel/ncc": "^0.34.0"
   }
 }
-
```
Comment (artemii-yanushevskyi) [inline package.json:20]: Removed by yarn

File: .yarnrc.yml [hunk 4]
```diff
@@ -0,0 +1 @@
+nodeLinker: node-modules
```
Comment (artemii-yanushevskyi) [inline .yarnrc.yml:1]: Added by yarn

**Review by mikhail-tokarev — APPROVED** [review]
🚀

**Pull request merged**